### PR TITLE
Hide worker containers on enterprise

### PR DESCRIPTION
**Pull request by cs278**
Fixes last remaining problem raised in #865 by disabling worker selection in enterprise environments.

### CODE REVIEW

File: src/Command/CommandBase.php [hunk 1]
```diff
@@ -885,6 +885,12 @@ protected function selectRemoteContainer(InputInterface $input, $includeWorkers
         }
 
         $environment = $this->getSelectedEnvironment();
+
+        // Enterprise environments do not have seperate containers for workers.
+        if ($includeWorkers && $environment->deployment_target === 'enterprise') {
```
Comment (pjcdawkins) [inline src/Command/CommandBase.php]: Unfortunately the deployment_target for an Enterprise (dedicated) environment isn't necessarily the string `enterprise`. Elsewhere we check for `!== 'local'`, which is hacky but works.

File: src/Command/CommandBase.php [hunk 2]
```diff
@@ -885,6 +885,12 @@ protected function selectRemoteContainer(InputInterface $input, $includeWorkers
         }
 
         $environment = $this->getSelectedEnvironment();
+
+        // Enterprise environments do not have seperate containers for workers.
```
Comment (pjcdawkins) [inline src/Command/CommandBase.php]:
```suggestion
        // Enterprise environments do not have separate containers for workers.
```

File: src/Command/CommandBase.php (hunk 1 above)
Comment (pjcdawkins) [reply src/Command/CommandBase.php]: For this reason - the slightly unreliable hack - I want to hide the interactive worker choice for non-`local` environments, but still respect the `--worker` option.

Comment (cs278) [reply src/Command/CommandBase.php]: @pjcdawkins updated, I haven't tested it however.

**Pull request merged**